vmdb/app/models/miq_provision_configured_system_request.rb

```diff
@@ -39,6 +39,10 @@ def self.new_request_task(attribs)
     request_task_class_from(attribs).new(attribs)
   end
 
+  def org_controller
```

Comment:
Can we choose a clearer name for this method?  Not clear what the org prefix stands for